AndrewTehan/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ENTER_STRING = 'Enter string: '
+ERROR_MESSAGE = "String can't be blank"
+EMPTY_STRING = ''
+EXIT = 'exit!'
+
+def run_cli
+  print ENTER_STRING
+  string = gets.chomp
+
+  case string
+  when EXIT then exit
+  when EMPTY_STRING then puts ERROR_MESSAGE
+  else puts get_middle(string)
+  end
+
+  run_cli
+end
+
+def return_by_index(string, index)
+  string[index]
+end
+
+def return_by_range(string, start_index, end_index)
+  string[start_index..end_index]
+end
+
+def get_middle(string)
+  size = string.size
+  middle_index = size / 2
+  size.odd? ? return_by_index(string, middle_index) : return_by_range(string, middle_index - 1, middle_index)
```

Comment:
I think of a bit easier and smaller solution like, what do you think about that 👇🏻 ?

```
def get_middle(string)
  middle_index = (string.size - 1) / 2.0
  string[middle_index.floor..middle_index.round]
end
```